paddle/fluid/platform/mkldnn_helper.h

```diff
@@ -237,5 +237,16 @@ class MKLDNNHandler {
   bool is_reusing_;
 };
 
+inline mkldnn::memory::format MKLDNNFormatForSize(
+    size_t dims_size, mkldnn::memory::format data_format) {
+  if (dims_size == 1) {
+    if (dims_size == 2) {
```

Comment:
Is that right? maybe forget else ?

```
dims_size == 1
              ? mkldnn::memory::format::x
              : dims_size == 2 ? mkldnn::memory::format::nc : default_format
```

Reply:
Could you reuse the code in  https://github.com/PaddlePaddle/Paddle/pull/11544/files#diff-70d521f84c13ef539b9ef82a8886b693 ?